openquake/calculators/event_based_risk.py

```diff
@@ -71,10 +78,13 @@ def event_based_risk(riskinputs, riskmodel, rlzs_assoc, monitor):
         a single array of dtype elt_dt, or an empty list
     """
     lti = riskmodel.lti  # loss type -> index
-    specific_assets = monitor.oqparam.specific_assets
-    result = cube(
-        monitor.num_outputs, len(lti), len(rlzs_assoc.realizations), list)
-    for out_by_rlz in riskmodel.gen_outputs(riskinputs, rlzs_assoc, monitor):
+    L, R = len(lti), len(rlzs_assoc.realizations)
```

Comment:
There is a long tradition in scientific computing about using one-letter variable for integers. Also, the names are used in all other calculators, so I want to keep them. `r` is the loop index in `range(R)`, `l` is the loop index in `range(L)`.